diff --git a/linux_os/guide/services/base/package_abrt_removed.rule b/linux_os/guide/services/base/package_abrt_removed.rule
new file mode 100644
index 0000000000..6cdca812c8
--- /dev/null
+++ b/linux_os/guide/services/base/package_abrt_removed.rule
@@ -0,0 +1,22 @@
+documentation_complete: true
+
+prodtype: rhel6,rhel7,fedora
+
+title: 'Uninstall Automatic Bug Reporting Tool (abrt)'
+
+description: |-
+    The Automatic Bug Reporting Tool (<tt>abrt</tt>) collects
+    and reports crash data when an application crash is detected. Using a variety
+    of plugins, abrt can email crash reports to system administrators, log crash
+    reports to files, or forward crash reports to a centralized issue tracking
+    system such as RHTSupport.
+    {{{ describe_package_remove(package="abrt") }}}
+
+rationale: |-
+    Mishandling crash data could expose sensitive information about
+    vulnerabilities in software executing on the system, as well as sensitive
+    information from within a process's address space or registers.
+
+severity: unknown
+
+{{{ complete_ocil_entry_package(package="abrt") }}}
diff --git a/rhel7/profiles/ospp42.profile b/rhel7/profiles/ospp42.profile
index 8550434ffa..376aebba51 100644
--- a/rhel7/profiles/ospp42.profile
+++ b/rhel7/profiles/ospp42.profile
@@ -184,3 +184,4 @@ selections:
     - audit_rules_etc_group_open
     - audit_rules_etc_group_openat
     - audit_rules_etc_group_open_by_handle_at
+    - package_abrt_removed

diff --git a/rhel7/profiles/ospp42.profile b/rhel7/profiles/ospp42.profile
index 376aebba51..d6068ea7f7 100644
--- a/rhel7/profiles/ospp42.profile
+++ b/rhel7/profiles/ospp42.profile
@@ -185,3 +185,4 @@ selections:
     - audit_rules_etc_group_openat
     - audit_rules_etc_group_open_by_handle_at
     - package_abrt_removed
+    - package_sendmail_removed
