diff --git a/RHEL/7/input/profiles/rht-ccp.xml b/RHEL/7/input/profiles/rht-ccp.xml
index e611421..5236ffa 100644
--- a/RHEL/7/input/profiles/rht-ccp.xml
+++ b/RHEL/7/input/profiles/rht-ccp.xml
@@ -18,7 +18,7 @@
 <refine-value idref="var_password_pam_ocredit" selector="2"/>
 <refine-value idref="var_password_pam_lcredit" selector="2"/>
 <refine-value idref="var_password_pam_difok" selector="3"/>
-<refine-value idref="var_password_history_retain_limit" selector="5"/>
+<refine-value idref="var_password_pam_unix_remember" selector="5"/>
 <refine-value idref="var_accounts_user_umask" selector="077"/>
 <refine-value idref="login_banner_text" selector="usgcb_default"/>
 
diff --git a/shared/fixes/bash/accounts_password_pam_unix_remember.sh b/shared/fixes/bash/accounts_password_pam_unix_remember.sh
index 04e0767..98aecef 100644
--- a/shared/fixes/bash/accounts_password_pam_unix_remember.sh
+++ b/shared/fixes/bash/accounts_password_pam_unix_remember.sh
@@ -4,5 +4,5 @@ populate var_password_pam_unix_remember
 if grep -q "remember=" /etc/pam.d/system-auth; then   
 	sed -i --follow-symlink "s/\(remember *= *\).*/\1$var_password_pam_unix_remember/" /etc/pam.d/system-auth
 else
-	sed -i --follow-symlink "/^password[\s]sufficient[\s]pam_unix.so/ s/$/ remember=$var_password_pam_unix_remember/" /etc/pam.d/system-auth
+	sed -i --follow-symlink "/^password[[:space:]]\+sufficient[[:space:]]\+pam_unix.so/ s/$/ remember=$var_password_pam_unix_remember/" /etc/pam.d/system-auth
 fi
